### ISSUE TITLE
Trying to install R with gpu support on windows.

### ISSUE THREAD

**Issue by firearasi** [dmlc/mxnet#1537]
I am trying to build R package with prebuild binaries(GPU). 
Mxnet with GPU is working python perfectly well but I am not familiar with Python.
When I use "R CMD INSTALL --no-multiarch R-Package", I get the following error:

executor.cc:1:0: sorry, unimplemented: 64-bit mode not compiled in
 /*!
 ^
make: *** [executor.o] 错误(error) 1
Warning: 'make -f "Makevars.win" -f "C:/PROGRA~1/R/R-32~1.3/etc/x64/Makeconf" -f "C:/PROGRA~1/R/R-32~1.3/share/make/winshlib.mk" SHLIB_LDFLAGS='$(SHLIB_CXXLDFLAGS)' SHLIB_LD='$(SHLIB_CXXLD)' SHLIB="mxnet.dll" WIN=64 TCLBIN=64 OBJECTS="executor.o export.o io.o kvstore.o mxnet.o ndarray.o symbol.o"'2
ERROR: compilation failed for package 'mxnet'
* removing 'C:/Users/firearasi/Documents/R/win-library/3.2/mxnet'
* restoring previous 'C:/Users/firearasi/Documents/R/win-library/3.2/mxnet'

Environment:
make: GnuWin g++:mingw

And my gripe(if I have to use python): why there is no easy and concise tutorials for python as for R, like "Neural Network with MXNet in Five Minutes" and "Handwritten Digits Classification Competition"?

Let's say, what is the python equivalent for the following toy model?

`
library(mxnet)
set.seed(2016)
n=1000000
a=rnorm(n,0,20)
b=rnorm(n,20,10)
data=data.frame(a,b,c=2*a-3*b+sin(a)*b+rnorm(n,0,2))
data=data.matrix(data)

train_index=1:round(n*0.8,0)
train.x=data[train_index,1:2]
train.y=data[train_index,3]
test.x=data[-train_index,1:2]
test.y=data[-train_index,3]

net=mx.symbol.Variable('data')
net=mx.symbol.FullyConnected(net,num.hidden=6)
net=mx.symbol.Activation(net,act.type='relu')

net=mx.symbol.FullyConnected(net,num.hidden=12)
net=mx.symbol.Activation(net,act.type='relu')

net=mx.symbol.FullyConnected(net,num.hidden=8)
net=mx.symbol.Activation(net,act.type='relu')

net=mx.symbol.FullyConnected(net,num.hidden=4)

net=mx.symbol.FullyConnected(net,num.hidden=1)
net=mx.symbol.LinearRegressionOutput(net)

devices=mx.cpu()
model <- mx.model.FeedForward.create(net, X=train.x, y=train.y,
                                     ctx=devices, num.round=1000, array.batch.size=100,
                                     learning.rate=2e-6, momentum=0.9, wd=0.00001,
                                     eval.metric=mx.metric.rmse,
                                     initializer=mx.init.uniform(0.06),
                                     epoch.end.callback=mx.callback.log.train.metric(100),
                                     eval.data = list(data=test.x,label=test.y))
                                     
pred=predict(model,test.x)
sqrt(mean(pred-test.y)^2)
`

**Comment by thirdwing** [comment]
Are you using the Rtools?

If you are using Rtools, which version?

**Comment by firearasi** [comment]
@thirdwing 
Rtools? No, maybe that's because I have not built an R package before:)
I will look into that.
May you explain to me a bit?

**Comment by thirdwing** [comment]
It is the toolchain used by R. You can find it from https://cran.r-project.org/bin/windows/Rtools/

Please use https://cran.r-project.org/bin/windows/Rtools/Rtools32.exe ,since there might be some issues in the latest one.

It includes MinGW-w64 and other tools needed to build R packages.

**Comment by firearasi** [comment]
@thirdwing 
It seems cool: a gcc and a make:)

BTW I also tried building mxnet on Ubuntu, but I just could not success fully install nvidia driver there...

**Comment by thirdwing** [comment]
If possible we **strongly** suggest you using Linux.

**Comment by firearasi** [comment]
@thirdwing 
I know I know.
But I am having horrible stories about nvidia driver or cuda's on Ubuntu

**Comment by firearasi** [comment]
@thirdwing 
And rtools worked like a charm! Now I have my long desired working package! Thanks!
I think this should be mentioned in the documents:)
